subworkflows/local/shortread_assembly.nf

```diff
@@ -0,0 +1,39 @@
+
+//MODULES
+include { MEGAHIT                                               } from '../../modules/nf-core/megahit/main'
+include { SPADES as METASPADES                                  } from '../../modules/nf-core/spades/main'
+
+workflow SHORTREAD_ASSEMBLY {
+    take:
+    ch_short_reads_grouped           // [ [meta] , fastq1, fastq2] (mandatory)
+    ch_short_reads_spades
+
+    main:
+    ch_versions = Channel.empty()
+    ch_assembled_contigs = Channel.empty()
+
+    if (!params.single_end && !params.skip_spades) {
+        METASPADES(ch_short_reads_spades.map { meta, reads -> [meta, reads, [], []] }, [], [])
+        ch_spades_assemblies = METASPADES.out.scaffolds.map { meta, assembly ->
+            def meta_new = meta + [assembler: 'SPAdes']
+            [meta_new, assembly]
+        }
+        ch_assembled_contigs = ch_assembled_contigs.mix(ch_spades_assemblies)
+        ch_versions = ch_versions.mix(METASPADES.out.versions.first())
+    }
+
+    if (!params.skip_megahit) {
+        MEGAHIT(ch_short_reads_grouped)
+        ch_megahit_assemblies = MEGAHIT.out.contigs.map { meta, assembly ->
+            def meta_new = meta + [assembler: 'MEGAHIT']
+            [meta_new, assembly]
+        }
+        ch_assembled_contigs = ch_assembled_contigs.mix(ch_megahit_assemblies)
+        ch_versions = ch_versions.mix(MEGAHIT.out.versions.first())
```

Comment:
Apparently there is now a MEGAHIT module for MultiQC, https://docs.seqera.io/multiqc/modules/megahit

So you should be able to take the `log` channel and mix that into multiQC here